tests/src/core/testqgsfield.cpp

```diff
@@ -399,8 +399,25 @@ void TestQgsField::displayString()
   customEnglish.setNumberOptions( QLocale::NumberOption::OmitGroupSeparator );
   QLocale::setDefault( customEnglish );
   QCOMPARE( doubleField.displayString( 5.005005 ), QString( "5.005" ) );
+<<<<<<< HEAD
```

Comment:
@SebastienPeillet mind having a look?

Reply:
the two lines in the `HEAD` part are in the `84563cb2c5`, so the `HEAD` part can be remove to only keep the `84563cb2c5` part !

Reply:
Actually no, I didn't notice it the first time but why the backport only apply changes on testqgsfield.cpp on not on qgsfield.cpp ... ? I think we can close this PR. I will do the backport  manually

Reply:
ok

Reply:
Done here #33877